src/bin/pg_autoctl/cli_create_drop_node.c

```diff
@@ -632,6 +639,8 @@ cli_drop_node(int argc, char **argv)
 	log_info("Removed the keeper from the monitor and removed the state file "
 			 "from disk.");
 
+	log_info("Node data is not deleted. You can also use \"destroy\" "
+			 "command to delete all resources belonging to the node");
```

Comment:
Negative log entries sound like a problem, or read like something should be done. It's not the case here. What about the following messages:

``` c
    log_warn("configuration file \"%s\" has been preserved", config.pathnames.config);\
    if (directory_exists(config.pgSetup.pgdata))
    {
        log_warn("Postgres Data Directory \"%s\" has been preserved", config.pgSetup.pgdata);
    }
    log_info("drop node keeps your data and setup safe, you can still run Postgres or re-join the pg_auto_failover cluster later");
    log_info("HINT: to completely remove your local Postgres instance and setup, consider the command pg_autoctl destroy");
```

We should refrain from the PGDATA related WARN message when the directory doesn't exist anyway, even though the `drop node` command itself is not deleting it. It could have been deleted by the user.

Reply:
ok